Gds/src/fprime_gds/executables/run_deployment.py

```diff
@@ -222,7 +222,7 @@ def kill_wait():
     print("[INFO] Waiting 5 seconds for {0} to start".format(name))
     time.sleep(5)
     proc.poll()
-    if proc.returncode is not None:
+    if proc.returncode:
```

Comment:
What is the rationale behind not running this code if it returns 0?  A zero may be a successful completion of the program, but it is still not running after 5 seconds.

Reply:
Thus, what was intended to be a long-running process has exited after only 5 seconds, which to me represents an error condition.

Reply:
Can you please explain None vs 0 to me? The lxterminal process gives a 0 which I know as the "successful" return code, I couldn't quite figure out how why different terminals where giving me None vs 0

Reply:
It isn't the terminal.  `0` means the process ended in success.  A non-zero integer means the process ended in failure.  However `None` means the process is still running.  We want the process to continue running, so `None` is the only correct value, as that means the terminal and thus the FSW is still running 5 seconds later.  Any exit 0 or non-zero before that means the FSW stopped running, and thus an error occurred.

Reply:
Ah, makes more sense, appreciate the explanation. The app is still running and continues to run in lxterminal when it returns 0 (using above change so it doesn't exit), not sure why it considers this a successful exit if its still running.

Reply:
This should definitely stay as is, lxterminal is just different.